docs/moduleCom4FlowPy.rst

```diff
@@ -1,78 +1,190 @@
+- ``CPUPercent`` ... if ``nCPU = auto`` the number of CPUs used will be calculated based on this percentage.
+
+in the ``avaframe/com4FlowPy/(local_)com4FlowPyCfg.ini`` you can set the following module specific options/parameters:
+
+i) general model parameters:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- ``alpha``: :math:`\alpha` - travel angle in :math:`^{\circ}` :math:`\ldots` controls the maximum runout along the path
+- ``exp``: Exponent controling the concentration of the routing flux and therefore lateral spreading behavior. :math:`exp = 1 \ldots` widespread process paths, :math:`exp \rightarrow \infty \ldots` very confied process paths (single flow direction).
+- ``flux_threshold``: minimal flux value that is still processed by the routing algorithm (limits model runtimes by stopping further model caluclation in cells with excessively small *flux* values; thus also influencing process spreading together with ``exp``)
+- ``max_z``: :math:`z_{\delta}^{max}\;[\rm{m}]` maximum kinetic energy height limit :math:`\ldots` sets a hard limit to the max. energy line height (see: :cite:`HoJaRuZi_2013`) :math:`\rightarrow` can roughly be interpreted as a limit to maximum process velocities using the conversion :math:`v_{max}=(2 g z_{\delta}^{max})^{(1/2)}` 
+
+ii) additional modules (forest, infrastructure)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- ``forest``: if set to ``True`` the runout calculation is performed with the *forest module* (a forest layer has to be provided)
+- ``infra``: if set to ``True`` the calculation is performend with the *backcalculation module* (an infrastructure layer has to be provided)
+
+iii) forest module parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. Note::
+  Forest modules and parameters are currently updated/developed; we will update the description of parameters accordingly
+
+  - ``forestModule``: if ``forest=True`` different forest modules ``[ForestFriction, ForestDetrainment, ForestFrictionLayer]`` can be selected.
+
+      - if ``forestModule in {ForestFriction, ForestDetrainment}``: *forest_layer* has to be scaled from 0 (no forest effect) to 1 (optimal forest effect).
+      - if ``forestModule = ForestFrictionLayer`` each cell of the provided *forest_layer* has to contain either an ``absolute`` or ``relative`` value for ``alpha``, which will be utilized. 
+
+  depending on choice of the ``forestModule`` the following parameters can be set:
+
+  - ``forestModule = ForestFriction``:
+      - ``maxAddedFrictionFor``: tba.
+      - ``minAddedFrictionFor``: tba.
+      - ``velThForFriction``:  tba.  
+
+  - ``forestModule = ForestDetrainment``:
+      - ``maxAddedFrictionFor``: tba.
+      - ``minAddedFrictionFor``: tba.
+      - ``velThForFriction``:  tba.
+      - ``maxDetrainmentFor``: tba.
+      - ``minDetrainmentFor``: tba.
+      - ``velThForDetrain``: tba.
```

Comment:
This is the only minor comment I have: are you able to put sensible range examples here? That might be helpful...

Reply:
Thx, I agree! 

- For now I updated the descriptions of these parameters 

- I plan to add another section "Model Parameterisation" to the description at some point, where we can report on parameters that have been used in different studies (e.g. avalanches, rockfall) - however, I have to compile these first ...

- This might then also be a first step towards providing sensible defaults for different processes (e.g. for a simplified GUI version)